packages/strapi/lib/middlewares/parser/index.js

```diff
@@ -37,14 +37,32 @@ module.exports = strapi => {
      * Initialize the hook
      */
     initialize() {
-      strapi.app.use((ctx, next) => {
+      strapi.app.use(async (ctx, next) => {
         // disable for graphql
         // TODO: find a better way later
-        if (ctx.url === '/graphql') return next();
-        return body({
-          patchKoa: true,
-          ...omit(strapi.config.middleware.settings.parser, 'queryStringParser'),
-        })(ctx, next);
+        if (ctx.url === '/graphql') {
+          return next();
+        }
+
+        try {
+          const res = await body({
+            patchKoa: true,
+            ...omit(strapi.config.middleware.settings.parser, 'queryStringParser'),
+          })(ctx, next);
+          return res;
+        } catch (e) {
+          if (e.message.includes('maxFileSize exceeded')) {
```

Comment:
formidable doesn't throw already a 413 ?

Reply:
No it was just throwing an error, resulting to an error 500